rfcs/models/infrastructure/relays.md

```diff
@@ -73,11 +72,32 @@ Once a driver is designed for a given DLT, it can be reused in any network built
 
 ## Relay Functional API
 
-_TBD: List API function specs exposed to application clients, relays, and drivers respectively. Message formats will be described in 'formats/communication'_
```

Comment:
Can you add a sentence here as intro saying something like what is deleted above, and with links? Like "Here we list..."